tests/sdl2_mixer.c

```diff
@@ -37,6 +37,8 @@ int main(int argc, char* argv[]){
         return -1;
     if (Mix_PlayChannel(-1, wave, 0) == -1)
         return -1;
+    if (Mix_Init(MIX_INIT_OGG) == -1)
+        return -1;
```

Comment:
Do this this test simply not work without this?

Reply:
Without this, the specific problem, namely that ogg support is not compiled into SDL2_Mixer, is never triggered. SDL2_Mixer initialises just fine without having ogg compiled in. This line of code forces an error if it is missing.

Reply:
But did the example just not play sound before?

Reply:
Oh, it's an example? I thought it was a test. The sound plays fine without this addition. Would you rather I create a new test specifically for ogg files?

Reply:
No it is a test.  I just wonder what the purpose of the `Mix_Init(MIX_INIT_OGG)` line is if the code can play an ogg file even without this line?  Is it just a way to verify that ogg support is built in?   And if this code could play ogg files before this change, then surely ogg support *was* builtin previously? 

I'm just trying to understand the state before this change.  We *could* play ogg files?  But this Mix_Init(MIX_INIT_OGG) would fail?

Reply:
I am sorry for the miscommunication. Let me try to clarify it:
- No, playing ogg files is impossible without `Mix_Init(MIX_INIT_OGG)`
- No, playing ogg files was impossible before, even with `Mix_Init(MIX_INIT_OGG)`, it would return -1. `SDL_GetError()` would then return a message saying that ogg support was not compiled in.
- Playing wave files without `Mix_Init(MIX_INIT_OGG)` has worked before and continues to work
- This test, while not containing ogg files, will error when ogg support is not compiled in.
- The changes made in this commit ensure that ogg support is compiled in and that this test succeeds in its new form.

Does this clarify things? If you have more questions, I'll be happy to answer them.

Reply:
Got it.  Sorry I thought this test played an ogg file.

Reply:
Sounds good, but please add a comment, "this should error because OGG support was not compiled in".